replica/backup.go

```diff
@@ -48,7 +48,7 @@ func NewRestore(snapshotName, replicaAddress string) *RestoreStatus {
 	return &RestoreStatus{
 		replicaAddress: replicaAddress,
 		SnapshotName:   snapshotName,
-		State:          ProgressStateIncomplete,
+		State:          ProgressStateInprogress,
```

Comment:
Capitalize the `p` in `InProgress` since it's a word. 

`ProgressStateInProgress`